gadgets/Makefile

```diff
@@ -48,6 +48,13 @@ build: $(GADGETS)
 
 readmes: $(GADGETS_README)
 	counter=10 ; for i in $(shell git grep '^sidebar_position:' -- .|cut -d: -f1|grep /README.mdx) ; do sed -i "s/^sidebar_position:.*/sidebar_position: $$counter/g" $$i ; counter=$$(($$counter + 10)) ; done
+	@# Create symlinks for READMEs if not exists
+	for i in $(GADGETS_README) ; do \
+		if [ ! -f $(ROOT_DIR)../docs/gadgets/$$(dirname $$i).mdx ] ; then \
+			ln -sf ../../gadgets/$$i $(ROOT_DIR)../docs/gadgets/$$(dirname $$i).mdx ; \
```

Comment:
This could be moved to the `$(GADGETS_README):` target line 80, so there would not need to be a for loop.

Reply:
Done.